gdal/ogr/ogrsf_frmts/shape/sbnsearch.c

```diff
@@ -1,12 +1,13 @@
+#define sanity_checks 1
+#define DEBUG_IO 1
 /******************************************************************************
- * $Id$
  *
  * Project:  Shapelib
  * Purpose:  Implementation of search in ESRI SBN spatial index.
- * Author:   Even Rouault, even dot rouault at spatialys.com
+ * Author:   Even Rouault, even dot rouault at mines dash paris dot org
```

Comment:
should be reverted here / updated in shapelib

Reply:
https://github.com/OSGeo/shapelib/commit/3e59b35009f81538c2f3014a20c7abfc7a964ba3